small_ha_cluster.md

```diff
@@ -368,7 +368,7 @@ You have PredictionIO in `~/aml` so edit ~/pio-aml/conf/pio-env.sh to have these
     PIO_STORAGE_SOURCES_ELASTICSEARCH_TYPE=elasticsearch
     PIO_STORAGE_SOURCES_ELASTICSEARCH_HOME=/usr/local/elasticsearch
     # The next line should match the ES cluster.name in ES config
-    PIO_STORAGE_SOURCES_ELASTICSEARCH_CLUSTERNAME=some-cluster-name
+    PIO_STORAGE_SOURCES_ELASTICSEARCH_CLUSTERNAME=some-cluster
```

Comment:
This should match the cluster.name in the ES config. I'll change both to "some-cluster-name"